app/mikane/src/app/pages/event-info/event-info.component.html

```diff
@@ -28,9 +46,50 @@
 	<app-loading-spinner></app-loading-spinner>
 }
 
+<ng-template #statusContent>
+	<button
+		mat-stroked-button
+		disabled
+		class="event-info-marker"
+	>
+		{{ event?.status.name }}
+	</button>
+	<div>
+		@if (event?.status.id === EventStatusType.ACTIVE) {
+			This event is currently <b>active</b>, allowing participants to add to or modify it as needed. Once the event is ready for settlement, it will transition to 'ready to settle' status.
+		}
+		@else if (event?.status.id === EventStatusType.READY_TO_SETTLE) {
+			This event is currently <b>ready to settle</b>. Once all participants have completed their payments, the event will be marked as settled.
+		}
+		@else {
+			This event is currently marked as <b>settled</b>, indicating that all participants have completed their payments, and the event has been finalized.
+		}
```

Comment:
@switch can also be used
```suggestion
		@switch (event?.status.id) {
		@case (EventStatusType.ACTIVE) {
			This event is currently <b>active</b>, allowing participants to add to or modify it as needed. Once the event is ready for settlement, it will transition to 'ready to settle' status.
		}
		@case (EventStatusType.READY_TO_SETTLE) {
			This event is currently <b>ready to settle</b>. Once all participants have completed their payments, the event will be marked as settled.
		}
		@default {
			This event is currently marked as <b>settled</b>, indicating that all participants have completed their payments, and the event has been finalized.
		}
		}
```